docs/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#77](https://github.com/TheRacetrack/racetrack/issues/77))
 - Plugins added to racetrack can now be downloaded via the CLI or the admin interface.
   ([#451](https://github.com/TheRacetrack/racetrack/issues/451))
+- Fix portfolio UI sorting bug.
```

Comment:
Let's keep it in a dedicated section according to https://keepachangelog.com/en/1.0.0/
```suggestion

### Fixed
- Fix portfolio UI sorting bug.
```